terminus/geometry/ecef.py

```diff
@@ -0,0 +1,87 @@
+"""
+Copyright (C) 2017 Open Source Robotics Foundation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import math
+import numpy as np
+import latlon
+import enu
+
+
+class Ecef(object):
+
+    def __init__(self, x, y, z):
+        self.x = x
+        self.y = y
+        self.z = z
+
+    @classmethod
+    def equatorial_radius(cls):
+        # Semi-major axis of the WGS84 spheroid (meters)
+        return 6378137.0
+
+    @classmethod
+    def polar_radius(cls):
+        # Semi-minor axis of the wgs84 spheroid (meters)
+        return 6356752.314245
+
+    @classmethod
+    def first_eccentricity_parameter(cls):
+        # https://en.wikipedia.org/wiki/Eccentricity_(mathematics)#Ellipses
+        return math.sqrt(1.0 - (Ecef.polar_radius() ** 2) / (Ecef.equatorial_radius() ** 2))
```

Comment:
AFAICS this value should be constant, right? Can you just return the constant and leave the code commented out as a reference? Like:

```
def first_eccentricity_parameter(cls):
￼    # https://en.wikipedia.org/wiki/Eccentricity_(mathematics)#Ellipses
￼    # As a reference, this constant was computed as math.sqrt(1.0 - (Ecef.polar_radius() ** 2) / (Ecef.equatorial_radius() ** 2))
    return N
```